src/session/session_api.c

```diff
@@ -2508,6 +2528,12 @@ __wt_open_session(WT_CONNECTION_IMPL *conn, WT_EVENT_HANDLER *event_handler, con
     /* Acquire a session. */
     WT_RET(__open_session(conn, event_handler, config, &session));
 
+    /* Set the default helper error */
+    char *init_err_msg;
+    WT_ERR(__wt_malloc(session, 26 * sizeof(char), &init_err_msg));
+    strcpy(init_err_msg, "Placeholder error message");
+    session->helper_err = (WT_HELPER_ERROR){0, 0, init_err_msg};
```

Comment:
Does this logic need to be up in `__wt_open_session`? It would fit better inside of `__open_session`.

Reply:
Ah you're right, will change that.

Reply:
Regarding the new name, I'm not sure either - Jeremy has suggested WT_ERROR_INFO, but it hasn't been finalized yet so I don't know whether or not it's going in this PR

Reply:
[Renamed WT_HELPER_ERROR to WT_ERROR_INFO](https://github.com/wiredtiger/wiredtiger/pull/11327/commits/0531dd63d99a6a37bad4085d1d5fd03fb9402987)
Renamed in this commit (got the go-ahead from Jeremy)